README.md

```diff
@@ -4,7 +4,7 @@ The VotingWorks in-person voting system.
 
 ## About
 
-Includes software for a [ballot-marking device (BMD)](./frontends/bmd), a
+Includes software for a [ballot-marking device (BMD)](./apps/mark/frontend), a
 [ballot activation system (BAS)](./frontends/bas), a
```

Comment:
this made me realize idk how to do multiline code suggestions, but here's what these should be:

- `./frontends/bsd` -> `./apps/central-scan/frontend`
- `./frontends/precinct-scanner` -> `./apps/scan/frontend`
- `./frontends/election-manager` -> `./apps/admin/frontend`

reference to ballot activation system should be removed.

sorry you ran into this and thanks for jumping on making it better!

Reply:
np, thanks for the updated paths!